lib/runtime-core/src/cache.rs

```diff
@@ -241,3 +241,5 @@ pub trait Cache {
 /// A unique ID generated from the version of Wasmer for use with cache versioning
 pub const WASMER_VERSION_HASH: &'static str =
     include_str!(concat!(env!("OUT_DIR"), "/wasmer_version_hash.txt"));
+
+pub const WASMER_VERSION: &'static str = env!("CARGO_PKG_VERSION");
```

Comment:
I think we're already reading this somewhere else in runtime-core `VERSION` in lib.rs;

We should reuse the constant, or define it in a shared `constants.rs` file and use it for both